bin/plugin/commands/performance.js

```diff
@@ -150,12 +150,17 @@ async function setUpGitBranch( branch, environmentDirectory ) {
  *
  * @param {string} testSuite                Name of the tests set.
  * @param {string} performanceTestDirectory Path to the performance tests' clone.
+ * @param {string} environmentDirectory     Path to the environment directory.
  *
  * @return {Promise<WPPerformanceResults>} Performance results for the branch.
  */
-async function runTestSuite( testSuite, performanceTestDirectory ) {
+async function runTestSuite(
+	testSuite,
+	performanceTestDirectory,
+	environmentDirectory
+) {
 	await runShellScript(
-		`npm run test-performance -- packages/e2e-tests/specs/performance/${ testSuite }.test.js`,
+		`ENVIRONMENT_DIRECTORY=${ environmentDirectory } npm run test-performance -- packages/e2e-tests/specs/performance/${ testSuite }.test.js`,
```

Comment:
Huh, maybe this won't work in Windows? 🤔

Reply:
Should maybe test it 😄

Reply:
I refactored it to use the official API of Node so I think it should be fine now.